tgui/packages/tgui/interfaces/ListInputWindow/ListInputModal.tsx

```diff
@@ -53,7 +53,7 @@ export const ListInputModal = (props: ListInputModalProps) => {
 
   const getSearchBar = () => {
     return searchBarVisible
-      ? document.getElementById('search_bar').getElementsByTagName('input')[0]
+      ? document.getElementById('search_bar')?.getElementsByTagName('input')[0]
       : null;
   };
```

Comment:
Minor, but `getElementsByTagName('input')[0]` returns `undefined` if no matches, rather than null.

Both are valid, of course, but prefer to stick to one "not found" return type if you can rather than the return type implicitly being `null | undefined`.

Consider: `return (searchBarVisible && document.getElementById('search_bar')?.getElementsByTagName('input')[0]) || null;`